### PR TITLE
add participant role and new time format

### PR DESCRIPTION
**Pull request by jiahaoyu6666**
### Description:
- Use participantRole to determine message direction;
- Adding new event types (rehydration.transcript.updated and rehydration.finished);
- Updating message sent/received timestamp format, there are two formats:
  - `Fri, Aug 28, 11:11 AM` (If the timestamp is not current day)
  - `9:50 AM` (if it’s the current day)

---

### Testing:
*Is the code unit tested?* [YES]

### Screenshot: Chat direction is correct
image

### Screenshot: `sentTime` is from today
image

### Screenshot: `sentTime` is from the other day. Manually change the `sentTime` of last transcript in `ChatTranscriptor.js` `render` function to test.
```
render() {
    if(this.props.transcript && this.props.transcript.length > 0) {
      this.props.transcript[this.props.transcript.length - 1].transportDetails.sentTime = 1661275983 // 8/23/2022, 10:33:03 AM;
    }
    .....
}
```
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: global-setup.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Set time zone to UTC for jest config
```
Comment (haomingli2020) [inline global-setup.js:1]: Why is this needed? Also this config will be overriden by real TZ out of local testing right?

**Review by haomingli2020 — COMMENTED** [review]
Can you provide testing details and screenshots for changes?

File: global-setup.js (hunk 1 above)
Comment (jiahaoyu6666) [reply global-setup.js:1]: This config is for jest unit test.
I have added the screenshot for manual testing.

**Review by mhiaror — APPROVED** [review]

**Review by haomingli2020 — APPROVED** [review]
🚢 LGTM
